### ISSUE TITLE
Player Controller

### ISSUE THREAD

**Issue by OlleReberg** [OlleKilgarrah/Roguelike#3]
Suggestion: 3 types of attacks; 1. Weapon standard attack. 2. Weapon special attack. 3. Special skill/spell(ranged weaponart).

Possibility of having an inventory to toggle?

**Comment by jackahlberg** [comment]
Yeah nice. An inventory might be strange to have in a roguelike which tends to provide random loot in each run. We probably want a fast paced game too as well. An inventory in between runs could work alright though.

**Comment by OlleReberg** [comment]
Yeah I mean if we would have it, it'd be limited and not really a core mechanic. It could maybe be more of a sidebar containing consumables?

**Comment by jackahlberg** [comment]
Maybe hotkeys which consumes potions?

**Comment by OlleReberg** [comment]
Yeah that's basically what I was thinking

**Comment by Kapten21** [comment]
If we want to have Weapon/armor progression maybe we could combine the loot dropped from a boss that you get to the hq in a container/bag to use on something to mod/buff your gear?

**Comment by OlleReberg** [comment]
Yeah, alternatively we could have something similar to Death's Door : https://miro.com/app/board/o9J_lxio8go=/
